apps/meteor/server/lib/oauth/updateOAuthServices.ts

```diff
@@ -117,14 +113,10 @@ export async function updateOAuthServices(): Promise<void> {
 			}
 
 			await LoginServiceConfiguration.createOrUpdateService(serviceKey, data);
-			void notifyOnLoginServiceConfigurationChangedByService(serviceKey);
 		} else {
 			const service = await LoginServiceConfiguration.findOneByService(serviceName, { projection: { _id: 1 } });
 			if (service?._id) {
-				const { deletedCount } = await LoginServiceConfiguration.removeService(service._id);
-				if (deletedCount > 0) {
-					void notifyOnLoginServiceConfigurationChanged({ _id: service._id }, 'removed');
```

Comment:
I'm wondering: couldn't we just completely remove `notifyOnLoginServiceConfigurationChanged` (its definition)? I see it's called in only one other place where we also call `LoginServiceConfiguration.removeService` ([here](https://github.com/RocketChat/Rocket.Chat/blob/bcacbb1ceec16f9d6c3ee10cd29d0f9af1156779/apps/meteor/app/meteor-accounts-saml/server/lib/settings.ts#L138)), so we may not need it there too :thinking:

Reply:
Removing this event probably breaks OAuth login, since the event is used to notify meteor that the login services have changed. 
We no longer rely on the client's loginServices collection to show the login buttons but AFAIK we still need them there for the login itself to work.

Reply:
The mentioned error (`No broker set to broadcast`) also seems to happen for `watch.settings` sometimes, so it looks to me this is a problem related to #32540 - with some operations that used to rely on DbWatchers and now trigger notifications being executed before the broker is ready.